demos/projects/ESPRESSIF/az-ca-recovery/README.md

```diff
@@ -72,59 +80,32 @@ Parameter | Value
  `WiFi SSID` | _{Your WiFi SSID}_
  `WiFi Password` | _{Your WiFi password}_
 
-
 Under menu item `Azure IoT middleware for FreeRTOS Main Task Configuration`, update the following configuration values:
 
 Parameter | Value
 ---------|----------
- `Use PnP in Azure Sample` | Enabled by default. Disable this option to build a simpler sample without Azure Plug-and-Play.
- `Azure IoT Hub FQDN` | _{Your Azure IoT Hub Host FQDN}_ (Unused if Device Provisioning is enabled below)
- `Azure IoT Device ID` | _{Your Azure IoT Hub device ID}_
- `Azure IoT Module ID` | _{Your Azure IoT Hub Module ID}_ (optional, specify module id if using a device module; else leave blank if not)
+ `Enable Device Provisioning Sample` | _{Check this option to enable DPS in the sample}_
+ `Azure Device Provisioning Service ID Scope` | _{Your ID scope value}_
+ `Azure Device Provisioning Service Registration ID` | _{Your Device Registration ID value}_
+ `[RECOVERY] Azure Device Provisioning Service ID Scope` | _{Your ID scope value for the recovery instance}_
+ `[RECOVERY] Azure Device Provisioning Service Registration ID.` | _{Your Device Registration ID value for the recovery instance}_
 
-Select your desired authentication method with the `Azure IoT Authentication Method () --->`. The default option is `Symmetric Key`:
+Select your desired authentication method with the `Azure IoT Authentication Method () --->` as `Symmetric Key`:
```

Comment:
Do we support all auth systems with recovery and operational DPS / Hub services in this sample?
(If not, we should update these to only what is supported.)

Reply:
No this is symmetric key only right now.